v4/source/service_terms.yaml

```diff
@@ -0,0 +1,35 @@
+  /terms_of_service:
+    get:
+      tags:
+      - terms of service
+      summary: Get latest terms of service
+      description: |
+        Get latest terms of service from the server
+        ##### Permissions
+        Must be authenticated.
+      responses:
+        '200':
+          description: Terms of service fetched successfully
+          schema:
+            $ref: '#/definitions/TermsOfService'
+        '400':
+          $ref: '#/responses/BadRequest'
+        '401':
+          $ref: '#/responses/Unauthorized'
+    post:
+      tags:
+      - terms of service
+      summary: Creates a new terms of service
+      description: |
+        Creates new terms of service
+        ##### Permissions
+        Must be authenticated.
```

Comment:
I don't think this is right. Shouldn't the user have `manage_system` permissions to create a ToS?

Reply:
Server PR - https://github.com/mattermost/mattermost-server/pull/9556